tasks/main.yml

```diff
@@ -8,12 +8,17 @@
   stat: path=/tmp/{{ elasticsearch_file_name }}
   register: elasticsearch_downloaded
 
+- name: Check if we've already installed elasticsearch
+  stat: path=/usr/share/elasticsearch/lib/elasticsearch-{{ elasticsearch_version }}.jar
+  register: elasticsearch_installed
+
 - name: Download ES deb
   get_url: url={{ elasticsearch_base_url }}/{{ elasticsearch_file_name }} dest=/tmp/{{ elasticsearch_file_name }}
   when: not elasticsearch_downloaded.stat.exists
```

Comment:
Should this check for `elasticsearch_installed.stat.exists` too? No reason to re-download a deleted deb if we've already installed it.